items.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE public.catalogitems
```

Comment:
Subtitute integer primary keys with serial, uniform with SQL file in test resources